www/script/project/rtBuilders.js

```diff
@@ -162,7 +179,7 @@ define(function (require) {
                         return b.toLowerCase() === branch_type.toLowerCase();
                     };
 
-                if (hideUnstable === true && ($.inArray(UNSTABLE_TAG, builderTags) > -1 || $.inArray(WIP_TAG, builderTags) > -1)) {
+                if (hideUnstable === true && (rtBuilders.hasUnstableTag(builderTags, branch_type) || $.inArray(WIP_TAG, builderTags) > -1)) {
```

Comment:
You can remove all mention of the WIP_TAG, we don't use it anymore (feel free to do a separate PR for that if you prefer)

Reply:
I create new task with low priority: https://jira.hq.unity3d.com/browse/CDS-1870